### PR TITLE
Use flags for cert auth enabled

### PR DESCRIPTION
**Pull request by tempoz**
Restack of buildbuddy-io/buildbuddy/pull/1779

---

**Version bump**: Minor

### CODE REVIEW

**Comment by tempoz** [comment]
Current dependencies on/for this PR:
* master
  * **PR #1864** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1864" target="_blank">Graphite</a>  (this PR)
    * **PR #1780** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1780" target="_blank">Graphite</a> 
      * **PR #1865** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1865" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1864?utm_source=stack-comment).

**Review by bduffany — APPROVED** [review]

**Comment by tempoz** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/1864?utm_source=stack-comment).


Job Reference: tNi5q82iqro6xKMPIA1x

**Pull request merged**
